### PR TITLE
Issue : No Bussiness Logic For GCP 

### PR DESCRIPTION
**Pull request by rajshah1**
1. File Structure Updated 

2. Bussiness Logic Using IBM Garage Method Updated 

3. AUTO Generator Also Added

### CODE REVIEW

**Comment by rajshah1** [comment]
Need More Clarrification Regarding Health Check  Needed

File: main/java/application/app/Application.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    @Bean
+    public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
+        return args -> {
+            String port = environment.getProperty("local.server.port");
+
+            System.out.println();
+            System.out.println("Server started - http://localhost:" + port + "/swagger-ui.html");
+        };
+    }
```
Comment (rajshah1) [inline main/java/application/app/Application.java:36]: No Need FOR This

File: main/java/com/ibm/cloud_garage/logging/LoggingContext.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.ibm.cloud_garage.logging;
+
```
Comment (rajshah1) [inline main/java/com/ibm/cloud_garage/logging/LoggingContext.java:2]: Remove Logging Changes

**Comment by rajshah1** [comment]
Please Review New Request

**Comment by Tarun-Student** [comment]
Solution looks Good Thanks .

Merging PR

**Pull request merged**